src/main/java/org/kuali/kfs/fp/document/web/struts/DisbursementVoucherAction.java

```diff
@@ -202,7 +203,7 @@ public ActionForward printDisbursementVoucherCoverSheet(ActionMapping mapping, A
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         DisbursementVoucherCoverSheetService coverSheetService = SpringContext.getBean(DisbursementVoucherCoverSheetService.class);
 
-        coverSheetService.generateDisbursementVoucherCoverSheet(directory, DisbursementVoucherConstants.DV_COVER_SHEET_TEMPLATE_NM, document, baos);
+        coverSheetService.generateDisbursementVoucherCoverSheet(document, baos);

```

Comment:
It looks like the local directory variable is not needed any longer, should that be removed?